src/main/scala/rocketchip/Configs.scala

```diff
@@ -27,17 +27,14 @@ class BasePlatformConfig extends Config(
       lazy val innerDataBeats = (8 * site(CacheBlockBytes)) / innerDataBits
       pname match {
         //Memory Parameters
-        case MIFTagBits => Dump("MIF_TAG_BITS", 5)
-        case MIFDataBits => Dump("MIF_DATA_BITS", 64)
-        case MIFAddrBits => Dump("MIF_ADDR_BITS",
```

Comment:
I think you aren't dumping MIF_ADDR_BITS anymore... Do we actually still need to export all the MEM_XX_BITS things via the dump? Isn't the test harness that uses them now defunct?